client/src/features/projectsV2/fields/NameFormField.tsx

```diff
@@ -51,9 +52,7 @@ export default function NameFormField<T extends FieldValues>({
         rules={{ required: true, maxLength: 99 }}
       />
       <div className="invalid-feedback">Please provide a name</div>
-      <FormText id={`${entityName}NameHelp`} className="input-hint">
-        The name you will use to refer to the {entityName}.
-      </FormText>
+      {helpText}
```

Comment:
You should keep the `<FormText>` component here, otherwise the association with the input field is broken.

```suggestion
      {helpText && <FormText id={`${entityName}-help`}>{helpText}</FormText>}
```